docs/source/templates.rst

```diff
@@ -53,6 +53,28 @@ The first line again indicates that this template extends an existing template b
 The second and last line indicate that the enclosed lines are to be placed in the *body* block of the base template.
 The third line is the actual command that we want to add and the fourth line ensures that the code provided by the base template within the body block is still added.
 
+.. note::
+
+    This custom HPC `class` (see code below) should be stored in its own `python` file (named `hpc_setup.py`) in the main directory and not in the `project.py` file.
+    Storing it in the `project.py` file will result in the following warning, but the code should still execute properly.
+    You will also need to `import hpc_setup` in the `project.py` file.
+
+.. code-block:: python
+
+    class HPC_name(DefaultSlurmEnvironment):
+        """Subclass of DefaultSlurmEnvironment for 'hpc_name' HPC."""
+
+        # Find the hostname for the HPC by using the 'hostname' command.
+        # In this case 'hostname' produced 'hpc_node-slurm-2\.hpc_name\.university_name\.edu'.
+        hostname_pattern = r"hpc_node-slurm-.\.hpc_name\.university_name\.edu"
+        # The 'template' file contains the custom instructions to the HPC submission script, which
+        # are stored in the 'templates' folder.
+        template = "hpc_name.sh"
+
+.. warning::
```

Comment:
Just quote the warning as part of the note. The "warning" here will format the text below as a "documentation warning" which should be used in contexts where we try to grab the user's attention.

Reply:
done